src/frameworks/CoreServices/include/AE/AE.h

```diff
@@ -6,4 +6,20 @@
 OSErr AECreateDesc(DescType typeCode, const void *dataPtr, Size dataSize, AEDesc *result);
 OSErr AEDisposeDesc(AEDesc *theAEDesc);
 
+enum : AEEventID {
+kAEOpenApplication = 'oapp',
+kAEReopenApplication = 'rapp',
+kAEOpenDocuments = 'odoc',
+kAEPrintDocuments = 'pdoc',
+kAEOpenContents = 'ocon',
+kAEQuitApplication = 'quit',
+kAEAnswer = 'ansr',
+kAEApplicationDied = 'obit',
+kAEShowPreferences = 'pref',
+};
+
+enum : DescType {
+    kCoreEventClass = 'aevt',
+};
+
```

Comment:
These declarations should be in `AppleEvents.h` instead. That's where they live in Apple's SDK.

Reply:
Does `AppleEvents.h` currently exist, or should I create it in `src/frameworks/include/AE/`?

Reply:
You can create one at `src/frameworks/include/AE`.

Reply:
I think I've done it correctly - let me know if anything else needs changed!